suzieq/db/parquet/pq_coalesce.py

```diff
@@ -94,10 +95,21 @@ def write_files(table: str, filelist: List[str], in_basedir: str,
     state.block_start = int(block_start.timestamp())
     state.block_end = int(block_end.timestamp())
     if filelist:
-        this_df = ds.dataset(source=filelist, partitioning='hive',
-                             partition_base_dir=in_basedir) \
-            .to_table() \
-            .to_pandas()
+        files_pervers = defaultdict(list)
+        for file in filelist:
+            sqversmatch = re.search(r'sqvers=([0-9]*.[0-9]*)', file)
```

Comment:
The pattern is quite loose, if we are only expecting numeric versions (which I believe is our case) we can just change it to:
```suggestion
            sqversmatch = re.search(r'sqvers=([0-9]+.[0-9]+)', file)
```